src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -91,7 +91,7 @@ public IntegerSlider()
         //If the value field in the slider has a number greater than
         //In32.Maxvalue (or MinValue), the value will be changed to Int32.MaxValue (or MinValue)
         //The value will be changed, but to update the UI, this property is overridden here. 
-        public override int Value
+        public override long Value
```

Comment:
ugh, this is an API break, what a new slider class that inherits from sliderBase<long> ?